cfme/utils/bz.py

```diff
@@ -90,26 +92,27 @@ def from_config(cls):
                    password=credentials.get(cred_key, {}).get("password"),
                    cookiefile=None,
                    tokenfile=None,
-                   product=bz_conf.get("bugzilla", {}).get("product"))
+                   product=bz_conf.get("bugzilla", {}).get("product"),
+                   config_options=bz_conf)
 
     @cached_property
     def bugzilla(self):
         return _Bugzilla(**self.__kwargs)
 
     @cached_property
     def loose(self):
-        return cfme_data.get("bugzilla", {}).get("loose", [])
+        return self.config_options.get("loose", [])
 
     @cached_property
     def open_states(self):
-        return cfme_data.get("bugzilla", {}).get("skip", set([]))
+        return self.config_options.get("skip", set([]))
 
     @cached_property
     def upstream_version(self):
         if self.default_product is not None:
             return self.default_product.latest_version
         else:
-            return Version(cfme_data.get("bugzilla", {}).get("upstream_version", "9.9"))
+            return Version(self.config_options.get("upstream_version", "9.9"))
```

Comment:
we have downstream 10.0. shouldn't  upstream be more than it ?

Reply:
wouldntit make sense to replace this with `Version.latest()` ??

Reply:
@izapolsk I think its going to be 5.10 vs 9.9, not 10.0 vs 9.9 - but aside from that, I'm going to test with Version.latest().

Reply:
oh, you're right. this is 9.9 not 5.9

Reply:
@RonnyPfannschmidt looks like `Version.latest().vstring` is appropriate - want to maintain the type of a string there and not a version object (though it has `__str__` implementation)